process/logsevents/serialize.go

```diff
@@ -215,19 +215,42 @@ func serializeRoleData(buffSlice *data.BufferSlice, rd *data.RoleData, role stri
 	meta := []byte(fmt.Sprintf(`{ "update" : {"_index": "%s", "_id" : "%s" } }%s`, index, rd.Token, "\n"))
 	var serializedDataStr string
 	if rd.Set {
+		codeToExecute := `	
```

Comment:
could leave a TODO to change this way everywhere in the project

Reply:
done